addons/sourcemod/translations/shop.phrases.txt

```diff
@@ -477,6 +477,22 @@
 		"en"		"{green}[Shop] {default}Your data is being loaded!"
 	}
 
+	"You Buy Sure"
+	{
+		"en"		"Are you sure to purchase this item?"
+	}
+
+	"You Sell Sure"
+	{
+		"en"		"Are you sure to sell this item?"
+	}
+
+	"confirm_luck"
+	{
+		"#format"	"{1:i}"
+		"en"		"Are you sure to spend {1} credits for Try a Luck?"
+	}
+
```

Comment:
Add russian translation to `addons/sourcemod/translations/ru/shop.phrases.txt`, please:
```
	"You Buy Sure"
	{
		"ru"		"Вы уверены что хотите купить этот предмет?"
	}

	"You Sell Sure"
	{
		"ru"		"Вы уверены что хотите продать этот предмет?"
	}

	"confirm_luck"
	{
		"#format"	"{1:i}"
		"ru"		"Вы уверены, что хотите испытать удачу за {1} кредитов?"
	}
```

Reply:
Done